kobo/conf.py

```diff
@@ -274,9 +289,15 @@ def _get_from_import(self):
             self._get_token(skip_newline=False)
             self._skip_commas(skip_newline=False)
 
-        imported_config = self.__class__(config_file_suffix=self._config_file_suffix, debug=self._debug)
+        imported_config = self.__class__(
+            config_file_suffix=self._config_file_suffix,
+            debug=self._debug,
+            global_variables={k:v for k,v in self.items() if k in self._global_variables},
```

Comment:
Dict comprehensions are not available in Python 2.6, which may or may not be a problem.

Reply:
I've changed that so it uses list generators which should be available in python-2.6.